src/openrct2/world/Banner.cpp

```diff
@@ -44,21 +44,17 @@ rct_banner gBanners[MAX_BANNERS];
  *
  *  rct2: 0x006B7EAB
  */
-static sint32 banner_get_ride_index_at(sint32 x, sint32 y, sint32 z)
+static uint8 banner_get_ride_index_at(sint32 x, sint32 y, sint32 z)
```

Comment:
Consider that we will expand the amount of rides possible in the near future, is returning the ride index as an uint8 really a good idea?

Reply:
I'd say that, until we do, it's the valid thing to do.

Reply:
If expanded in the future, it _should_ generate a warning, though now I think about it I do believe these warnings are disabled. It'd probably be nicest to have `ride_index` and `banner_index` types.

Reply:
I've added the new `BannerIndex` type and use it in all the places I could find, but left out `RideIndex` for now, as it's out of scope for this PR.